modules/migration/migration-resources/5.8.0/dbscripts/step1/identity/mysql.sql

```diff
@@ -18,13 +24,12 @@ CREATE TABLE IF NOT EXISTS IDN_AUTH_USER_SESSION_MAPPING (
 	SESSION_ID VARCHAR(255) NOT NULL,
 	CONSTRAINT USER_SESSION_STORE_CONSTRAINT UNIQUE (USER_ID, SESSION_ID));
 
-CREATE INDEX IDX_USER_ID ON IDN_AUTH_USER_SESSION_MAPPING (USER_ID);
+CALL skip_index_if_exists('IDX_USER_ID', 'IDN_AUTH_USER_SESSION_MAPPING', '(USER_ID)');
 
-CREATE INDEX IDX_SESSION_ID ON IDN_AUTH_USER_SESSION_MAPPING (SESSION_ID);
+CALL skip_index_if_exists('IDX_SESSION_ID', 'IDN_AUTH_USER_SESSION_MAPPING', '(SESSION_ID)');
 
 CALL skip_index_if_exists('IDX_OCA_UM_TID_UD_APN','IDN_OAUTH_CONSUMER_APPS','(USERNAME,TENANT_ID,USER_DOMAIN, APP_NAME)');
 
 CALL skip_index_if_exists('IDX_SPI_APP','SP_INBOUND_AUTH','(APP_ID)');
 
 CALL skip_index_if_exists('IDX_IOP_TID_CK','IDN_OIDC_PROPERTY','(TENANT_ID,CONSUMER_KEY)');
```

Comment:
Shall we drop the stored procedures we create from the script at the end so that it will not leave any remainders after the migration is completed?

Reply:
Fixed in 1457380ebc6189fa1dca783f9aab9f093f155812